x-pack/legacy/plugins/actions/README.md

```diff
@@ -247,6 +248,27 @@ This action type interfaces with the [Slack Incoming Webhooks feature](https://a
 |---|---|---|
 |message|the message text|string|
 
+
+## index, action id: `.index`
+
+The config and params properties are modelled after the [Watcher Index Action](https://www.elastic.co/guide/en/elastic-stack-overview/master/actions-index.html).  The index can be set in the config or params, and if set in config, then the index set in the params will be ignored.
+
+#### config properties
+
+|Property|Description|Type|
+|---|---|---|
+|index|The Elasticsearch index to index into.|string _(optional)_|
+
+#### params properties
+
+|Property|Description|Type|
+|---|---|---|
+|index|The Elasticsearch index to index into.|string _(optional)_|
+|doc_id|The optional _id of the document.|string _(optional)_|
+|execution_time_field|The field that will store/index the action execution time.|string _(optional)_|
```

Comment:
I'm seeing some inconsistencies in the casing for params. In slack we have camel case (`webhookUrl`) but for index we now introduce snake case (`execution_time_field`). I think we should pick one.

Reply:
++ on consistency here

Reply:
Ya, Kibana APIs seems to use camelCase almost all the time for JS properties; will switch.

Reply:
changed to camelCase